src/spikeinterface/benchmark/benchmark_merging.py

```diff
@@ -0,0 +1,158 @@
+from __future__ import annotations
+
+from spikeinterface.sortingcomponents.merging import merge_spikes
+from spikeinterface.comparison import compare_sorter_to_ground_truth
+from spikeinterface.widgets import (
+    plot_agreement_matrix,
+    plot_unit_templates,
+    plot_amplitudes,
+    plot_crosscorrelograms,
+)
+
+import numpy as np
+from .benchmark_base import Benchmark, BenchmarkStudy
+
+
+class MergingBenchmark(Benchmark):
+
+    def __init__(self, recording, splitted_sorting, params, gt_sorting, splitted_cells=None):
+        self.recording = recording
+        self.splitted_sorting = splitted_sorting
+        self.method = params["method"]
+        self.gt_sorting = gt_sorting
+        self.splitted_cells = splitted_cells
+        self.method_kwargs = params["method_kwargs"]
+        self.result = {}
+
+    def run(self, **job_kwargs):
+        self.result["sorting"], self.result["merges"], self.result["outs"] = merge_spikes(
```

Comment:
I can not, because I want to use a higher level function here that can launch iteratively compute_merge_unit_groups. So only think I can do is to can modify auto_merge_units into auto_merge_units_iterative (and the two will be unified), and use this function in the benchmark